integration-test/src/test/java/org/apache/iotdb/session/it/IoTDBSessionSchemaTemplateIT.java

```diff
@@ -54,11 +56,15 @@
 import static org.junit.Assert.fail;
 
 @RunWith(IoTDBTestRunner.class)
```

Comment:
seems this redundant runner has led to the CI failure.

Reply:
Fixed.